packages/app/src/cli/services/logs.ts

```diff
@@ -41,6 +42,7 @@ export async function logs(commandOptions: LogsOptions) {
     filters,
   }
 
+  renderText({text: 'Waiting for app logs...'})
```

Comment:
To ensure compatibility with JSON parsing, we'll need to print this differently when the `json` format is in use so that we can support tools like jq:
```
MacBook-Pro:cli duncanuszkay$ echo "Waiting for app logs..." | jq
parse error: Invalid numeric literal at line 1, column 8
MacBook-Pro:cli duncanuszkay$ echo "{ \"a\": \"Waiting for app logs...
\" }" | jq
{
  "a": "Waiting for app logs..."
}
```

Reply:
Ah! got it, thanks Duncan!